chapters/planning-data-work.tex

```diff
@@ -1,147 +1,459 @@
+Most syncing services offer some kind of rudimentary version control;
+These are usually enough to manage changes to binary files (such as Word and PowerPoint documents)
+without needing to rely on dreaded filename-based versioning conventions.
+For technical files, however, a more detailed version control system is usually desirable.
+We recommend using Git\sidenote{
+  \textbf{Git:} a multi-user version control system for collaborating on and tracking changes to code as it is written.}
+for all plaintext files.
+Git tracks all the changes you make to your code,
+and allows you to go back to previous versions without losing the information on changes made.
+It also makes it possible to work on multiple parallel versions of the code,
+so you don't risk breaking the code for other team members as you try something new.
+The DIME file management and organization approach is designed with this in mind.
+
+Once the \texttt{DataWork} folder's directory structure is set up,
+you should adopt a file naming convention.
+You will generally be working with two types of files:
+``technical'' files, which are those that are accessed by code processes,
+and ``non-technical'' files, which will not be accessed by code processes.
+The former takes precedent: an Excel file is a technical file
+even if it is a field log, because at some point it will be used by code.
+We will not give much emphasis to non-technical files here;
+but you should make sure to name them in an orderly fashion that works for your team.
+These rules will ensure you can find files within folders
+and reduce the amount of time others will spend opening files
+to find out what is inside them.
+Some of the differences between the two file types are major and may be new to you. For example,
+you can use spaces and datestamps in names of non-technical files, but not technical ones:
+the former might have a name like \texttt{2019-10-30 Sampling Procedure Description.docx}
+while the latter might have a name like \texttt{endline-sampling.do}.
+Technical files have stricter requirements than non-technical ones.\sidenote{
+  \url{http://www2.stat.duke.edu/~rcs46/lectures_2015/01-markdown-Git/slides/naming-slides/naming-slides.pdf}}
+For example, you should never use spaces in technical names;
+this can cause problems in code. (This includes all folder names.)
+Similarly, technical files should never include capital letters.
+One organizational practice that takes some getting used to
+is the fact that the best names from a coding perspective
+are usually the opposite of those from an English perspective.
+For example, for a deidentified household dataset from the baseline round,
+you should prefer a name like \texttt{baseline-household-deidentified.dta},
+rather than the opposite way around as occurs in natural language.
+This ensures that all \texttt{baseline} data stays together,
+then all \texttt{baseline-household} data,
+and finally provides unique information about this specific file.
```

Comment:
New paragraph:

> We will not give much emphasis to files that are not linked to code here;
> but you should make sure to name them in an orderly fashion that works for your team.
> These rules will ensure you can find files within folders
> and reduce the amount of time others will spend opening files
> to find out what is inside them.
> The main point to be considered is that files accessed by code need face more restrictions\sidenote{
>   \url{http://www2.stat.duke.edu/~rcs46/lectures_2015/01-markdown-Git/slides/naming-slides/naming-slides.pdf}},
> as different software and operating systems read file names in different ways.
> Some of the differences between the two naming approaches are major and may be new to you, 
> so below are a few examples. 
> Introducing spaces between words in a file name (including the folder path) 
> can break a file's path when it's read by code, 
> so while a Word document may be called \texttt{2019-10-30 Sampling Procedure Description.docx},
> a related do file would have a name like \texttt{endline-sampling.do}.
> Adding timestamps to binary files as in the example above can be useful,
> as it is less straight-forward to track changes. 
> However, for plaintext files tracked using Git, timestamps are an unnecessary distraction.
> Similarly, technical files should never include capital letters, 
> as strings are not case-sensitive in some software.
> Finally, one organizational practice that takes some getting used to
> is the fact that the best names from a coding perspective
> are usually the opposite of those from an English perspective.
> For example, for a deidentified household dataset from the baseline round,
> you should prefer a name like \texttt{baseline-household-deidentified.dta},
> rather than the opposite way around as occurs in natural language.
> This ensures that all \texttt{baseline} data stays together,
> then all \texttt{baseline-household} data,
> and finally provides unique information about this specific file.

Reply:
@luizaandrade , are you putting in quote formatting what you are suggesting that this paragraph should be instead? it is unclear to me

Reply:
Yes, that's right. I see it is confusing

Reply:
Inserted in 35d91c3e22fd67afa2323ed9200f18377614ce96